src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -2154,4 +2154,47 @@ void TestReturnAirPathIntegrity(OutputFiles &outputFiles, bool &ErrFound, Array2
     }
 }
 
+void CalcTotalSensibleLatentOutput(Real64 const MassFlow,  // air mass flow rate, {kg/s}
+                                   Real64 const TDB2,      // dry-bulb temperature at state 2 {C}
+                                   Real64 const dW2,       // humidity ratio at  at state 2
```

Comment:
@Nigusse Argument name consistency. If TDB2 is used, W2 should be used, instead of dW2. Same as dW1. In general, dW may be understood as W difference between two pints.